ebi_eva_common_pyutils/assembly/assembly.py

```diff
@@ -29,3 +30,17 @@ def get_supported_asm_from_ensembl(tax_id: int) -> str:
     if assembly_accession_attribute in response:
         return str(response.get(assembly_accession_attribute))
     return None
+
+
+@lru_cache
+def get_ensembl_rapid_release_data():
+    list_data = json_request('https://ftp.ensembl.org/pub/rapid-release/species_metadata.json')
```

Comment:
Looking at the JSON file, it seems to contains an annotated list of assembly. This means that there is more than one assembly per taxonomy. I think we need to think about how we will select an assembly over another one.

Reply:
Clearly I did not check this JSON very carefully... I guess there's a `release-date` attribute that we can use to select the most recent one.

Reply:
Actually it's more complicated than that... there are a number of instances where they have multiple assemblies for a taxonomy with the same release date, just associated with different strains.